shop/src/themes/shared/checkout/Information.js

```diff
@@ -30,6 +32,38 @@ export const Information = () => {
           return
         }
         dispatch({ type: 'updateUserInfo', info: newState })
+
+        //Calculate taxes
```

Comment:
As opposed to duplicate the same code in the Information.js and Shipping.js files, could we extract it and put it in a common file that then gets imported by both?

Reply:
https://github.com/OriginProtocol/dshop/pull/1013/commits/14cabd3357fa81d05ee2021565ccdcd2521220cc